errata/test.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
```

Comment:
Do we need the file "errata/test.html"? It seems we can avoid providing it. Am I missing something?

Reply:
No, we can remove it.

Reply:
Done.